common/metrics/defs.go

```diff
@@ -625,6 +626,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		LeaseFailureCounter:           {metricName: "lease.failures"},
 		ConditionFailedErrorCounter:   {metricName: "condition-failed-errors"},
 		RespondQueryTaskFailedCounter: {metricName: "respond-query-failed"},
+		ThrottleErrorCounter:          {metricName: "throttle.errors"},
```

Comment:
I think we shouldn't treat these like errors. What about throttle.empty-polls?

Reply:
Empty polls sounds misleading. It's useful to differentiate between truly empty where no error is returned versus throttling limit is reached. I think it's still an error, but one we might not want in the same monitoring classification. On server side, this should not cause any tickets but users should be able to set alerts on these. There could be potential issues if there are spikes in throttling errors.